CHANGES.rst

```diff
@@ -5,6 +5,8 @@ Changelog
 Unreleased
 ----------
 
+* The parameter ``node.attr.zone`` can now be overridden if required.
```

Comment:
Sounds good. I'd be a bit more specific, though. When the operator was run with the `CLOUD_PROVIDER` setting set, the setting was overridden by the operator, ignoring the cluster or node settings. So, maybe something along these lines:

```rst
* Fixed an inconsistent behavior where the configuration option
  :envvar:`CLOUD_PROVIDER` would override an explicitly defined
  ``node.attr.zone`` in either ``.spec.cluster.settings``,
  ``.spec.nodes.master.settings``, or ``.spec.nodes.data.*.settings``.
```